packages/playground/src/types/index.ts

```diff
@@ -15,6 +15,14 @@ export type InputFilterType = {
   type: string;
 };
 
+export enum NetworkFeatures {
+  ipv4 = "IPv4",
+  ip = "IP",
```

Comment:
![image](https://github.com/user-attachments/assets/e9de6d27-fa56-47fc-a820-b93de1aae683)
> this shown if the ipv4 selected 

i suggest to remove the ipv4 and always shows the ip only